src/content/apis/travel/skyscanner.md

```diff
@@ -0,0 +1,10 @@
+---
+name: SkyScanner
+description: Skyscanner API you can search for Skyscanner flights, hotels and car rentals.

```

Comment:
Sorry but on double checking the description looks grammatically incorrect, please fix it.